spec/lib/vet360/contact_information/service_spec.rb

```diff
@@ -93,6 +93,34 @@
     end
   end
 
+  describe '#get_telephone_transaction_status' do
+    let(:transaction_id) { 'd47b3d96-9ddd-42be-ac57-8e564aa38029' }
+    let(:transaction) { Vet360::Models::Transaction.new(id: transaction_id) }
```

Comment:
No concerns but I'm curious about this idiom... what's the difference between this and: `let(:transaction) { Vet360::Models::Transaction.new(id: 'd47b3d96-9ddd-42be-ac57-8e564aa38029') }`. I guess I'm asking what `let(:transaction_id) { 'd47b3d96-9ddd-42be-ac57-8e564aa38029' }` is doing here.

Reply:
Nothing in this situation, but in my next PR i'm going to be adding an expect for the actual transaction id so I figured I'd leave it broken out so I can compare the values.